src/modules/fstab/main.py

```diff
@@ -264,6 +280,10 @@ def generate_fstab_line_info(self, partition):
             options = "subvol={},".format(partition["subvol"]) + options
 
         if has_luks:
+            # Check if user mounted a previously encrypted partition
```

Comment:
From only the context here I don't see the match between the comment and the code that follows. In any case, `has_luks` is set:

```
has_luks = "luksMapperName" in partition
```
.. aha, so this check is for *luksMapperName* being set, but empty. Perhaps move the check earlier in the function?  Before `options` is set, that's where most of the sanity-checks are.